modules/router/src/Router.ts

```diff
@@ -147,6 +153,7 @@ export default class ConduitDefaultRouter extends ManagedModule<Config> {
     }
     await this.highAvailability();
     this.updateHealth(HealthCheckStatus.SERVING);
+    this.grpcSdk.createModuleClient('router', process.env.SERVICE_IP!);
```

Comment:
What would this accomplish? The Router's own `grpc-sdk` shouldn't be communicating with itself through gRPC.